src/shared/services/MirrorNodeService/types.ts

```diff
@@ -58,7 +58,7 @@ interface MirrorNodeAccountById {
 }
 
 interface MirrorNodeTokenPairResponse {
-  data: { contract_id: string };
+  data: { contract_id: string; evm_address: string };
 }
 
 interface MirrorNodeTokenBalance {
```

Comment:
I ran the app for testing some scenarios please find them below

1. In Proposal Type Page I found a difference in UX
<img width="734" alt="Screenshot 2023-09-27 at 9 07 38 AM" src="https://github.com/hashgraph/hedera-accelerator-defi-dex-ui/assets/89677167/470a795a-4062-43fc-85ba-37faf71cb7be">
<img width="759" alt="Screenshot 2023-09-27 at 9 07 43 AM" src="https://github.com/hashgraph/hedera-accelerator-defi-dex-ui/assets/89677167/562cc88a-df6d-4298-a1ba-d7de5adcb25a">

2. In Proposal Details Page I found this, hyperlink was not opening in hashscan
<img width="1155" alt="Screenshot 2023-09-27 at 9 15 22 AM" src="https://github.com/hashgraph/hedera-accelerator-defi-dex-ui/assets/89677167/fed11835-a755-42c1-9721-498354856040">

3. When we load the DAO details for the first time we see “details not found” for split seconds is this expected.?
4. If we are just using daoSafeIdQueryResults.data?.data.contract_id for new hook we could directly destructure it from mirror node and use directly in components
5. We can keep “0.0.0” Hbar accout id representation in constants.

Reply:
@anshulkhare-lab49 ^Have all of these comments been addressed?